versions.json

```diff
@@ -0,0 +1,21 @@
+{
+    "tools": {
+        "kubectl": [
+            "1.31.1",
+            "1.30.5",
+            "1.29.9",
+            "1.28.14"
+        ],
+        "helm": [
+            "3.16.1"
+        ],
+        "powershell": [
+            "7.4.5"
+        ]
+    },
+    "latest": "1.30",
+    "deprecatedKubectlVersions": {
+        "1.26": "1.26",
```

Comment:
Is it worth making this json structure a bit more explicit about what the value means in this case?

e.g. 
"1.26": {latestRevision: "<some_hash>"}
or
"1.26": {latestImageTag: "1.26-<some_hash>"}

At the moment it feels unclear when looking at it without prior context.

Reply:
yeah, fair. 👀